pkg/sdk/poc/main.go

```diff
@@ -10,6 +10,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/genhelpers"
```

Comment:
why this import is added here?

Reply:
Same reason as above (it's used at the bottom of this file).